src/main/scripts/create-ds.sh

```diff
@@ -18,13 +18,14 @@
 wasRootPath=$1                                      # Root path of WebSphere
 wasProfileName=$2                                   # WAS profile name
 wasClusterName=$3                                   # WAS cluster name
-dbType=$4                                           # Supported database types: db2
+dbType=$4                                           # Supported database types: [db2, oracle]
 jdbcDataSourceName=$5                               # JDBC Datasource name
 jdbcDSJNDIName=$(echo "${6}" | base64 -d)           # JDBC Datasource JNDI name
-dsConnectionURL=$(echo "${7}" | base64 -d)          # JDBC Datasource connection String
+dsConnectionString=$(echo "${7}" | base64 -d)       # JDBC Datasource connection String
```

Comment:
Same story about the positional parameters.

Reply:
Env vars are supported by Microsoft.Resources/deploymentScripts but not supported for Microsoft.Compute/virtualMachines/extensions as for now.

Reply:
I accept that fact. Might it make sense to record a backlog item for using `deploymentScripts` instead of `extensions` here?

Reply:
For this case, I prefer to use VM extension as all these scripts/commands are running inside VM/Operation System. If we switch to `deploymentScripts`, we have to remotely connect to VM (e.g., using `ssh`) to execute these post-deployment scripts which seems lower performance and has higher requirement of connectivity stability.